x-pack/plugins/apm/public/components/alerting/error_count_alert_trigger/index.tsx

```diff
@@ -64,14 +86,14 @@ export function ErrorCountAlertTrigger(props: Props) {
       unit={i18n.translate('xpack.apm.errorCountAlertTrigger.errors', {
         defaultMessage: ' errors',
       })}
-      onChange={(value) => setAlertParams('threshold', value)}
+      onChange={(value) => setAlertParams('threshold', value || 0)}
     />,
     <ForLastExpression
-      onChangeWindowSize={(windowSize) =>
-        setAlertParams('windowSize', windowSize || '')
+      onChangeWindowSize={(timeWindowSize) =>
+        setAlertParams('windowSize', timeWindowSize || '')
       }
-      onChangeWindowUnit={(windowUnit) =>
-        setAlertParams('windowUnit', windowUnit)
+      onChangeWindowUnit={(timeWindowUnit) =>
+        setAlertParams('windowUnit', timeWindowUnit)
```

Comment:
Why these changes?

Reply:
It's to not shadow the `windowUnit` variable above and match the argument variable name that's in the other files that have a similar control.